frontend/src/modules/scribble/components/Canvas.vue

```diff
@@ -16,6 +17,12 @@ export default {
         defaultBrushStyle: Object,
     },
 
+    computed: {
+        ...mapState('scribble', {
+            isCanvasLocked: state => state.isCanvasLocked,
```

Comment:
will this load dynamically? like if someone changed it later in time after the "computed" lifecycle, what happens?